universal-login-react/src/core/services/DisconnectAccountService.ts

```diff
@@ -13,30 +13,30 @@ interface InputsType {
 
 const checkInputsAgainstError = (usernameLogged: string, inputs: InputsType) => ({
   usernameError: inputs.username !== usernameLogged,
-  verifyFieldError: inputs.verifyField !== 'DELETE MY ACCOUNT',
+  verifyFieldError: inputs.verifyField !== 'DISCONNECT',
```

Comment:
Pls, make it const and pls, use it in tests

Reply:
👍

Reply:
✔️